prefs.js

```diff
@@ -91,9 +92,16 @@ const App = new Lang.Class({
         this.field_display_mode = new Gtk.ComboBox({
             model: this._create_display_mode_options()});
 
-        let rendererText = new Gtk.CellRendererText();
-        this.field_display_mode.pack_start (rendererText, false);
-        this.field_display_mode.add_attribute (rendererText, "text", 0);
+        let rendererDsiplayModeText = new Gtk.CellRendererText();
+        this.field_display_mode.pack_start (rendererDsiplayModeText, false);
+        this.field_display_mode.add_attribute (rendererDsiplayModeText, "text", 0);
```

Comment:
Dsiplay ? Your fingers seems to have their own spelling :wink: : 
```suggestion
        let rendererDisplayModeText = new Gtk.CellRendererText();
        this.field_display_mode.pack_start (rendererDisplayModeText, false);
        this.field_display_mode.add_attribute (rendererDisplayModeText, "text", 0);
```